apps/etterlatte-saksbehandling-ui/client/src/components/oppgavebenken/OppgaveKolonner.tsx

```diff
@@ -7,13 +7,21 @@ import {
   soeknadTypeFilter,
   statusFilter,
   StatusFilter,
+  enhetFilter,
+  EnhetFilter,
 } from './typer/oppgavebenken'
 import { format } from 'date-fns'
 import { Tag } from '@navikt/ds-react'
 import HandlingerKnapp from './handlinger/HandlingerKnapp'
 import BrukeroversiktLenke from './handlinger/BrukeroversiktKnapp'
 import { tagColors } from '~shared/Tags'
 
+const mapEnhetNavn = (enhet?: EnhetFilter): string => {
+  const navn = (enhet ? enhetFilter[enhet as EnhetFilter]?.navn : undefined) ?? 'Ukjent'
```

Comment:
Hva skal til for at det blir "Ukjent"? (Hva er evt. forskjell på "Ukjent" og "Ingen"?

Reply:
Ingen er at vi får null tilbake fra backend.

Ukjent er fallback som er brukt standard i de andre filtere hvis vi får en verdi som vi ikke vet hva er.